timesketch/lib/sigma_util.py

```diff
@@ -39,24 +39,25 @@ def get_sigma_config_file():
         ValueError: If SIGMA_CONFIG is not found in the config file.
             or the Sigma config file is not readabale.
     """
-    config_file = current_app.config.get('SIGMA_CONFIG')
+    if config_file is None:
+        config_file_path = current_app.config.get('SIGMA_CONFIG')
 
     if not config_file:
         raise ValueError(
             'SIGMA_CONFIG not found in config file')
 
-    if not os.path.isfile(config_file):
+    if not os.path.isfile(config_file_path):
```

Comment:
config_file_path is not set if `config_file` is set...

so ou'll need to pick one

```
if config_file:
  config_file_path = config_file
else:
  config_file_path = current_app.conf....
```

something like that

Reply:
indeed, also mentioned the optional arg in the docstring, forgot that.